buildSrc/src/main/groovy/com/github/okbuilds/okbuck/generator/BuckFileGenerator.groovy

```diff
@@ -60,8 +72,6 @@ final class BuckFileGenerator {
             }
         }
 
-        createProjectConfigRule(project, project.okbuck.projectTargets)
```

Comment:
we are not using the return value here, it's already being called above - https://github.com/OkBuilds/OkBuck/pull/136/files#diff-8e0e8d98076e9ca8aa632421f239491eL30